### ISSUE TITLE
Assert look likes doesn't work

### ISSUE THREAD

**Issue by anhvuong123** [Codeception/CodeceptJS#786]
I've defined a basic case to start working with Assert in CodeceptJS:
```
var assert = require('assert');
assert.equal('true', 'false');
```
Assert looks like doesn't work. The test consists code above was passed without alert relating to Fail Assert. Please help me via the case.

Thanks,

**Comment by alexashley** [comment]
@anhvuong123 To my knowledge, Codecept doesn't ship with an assertion library, the examples use Node's [assert](https://nodejs.org/api/assert.html).

Also your example works as expected for me in the Node REPL:
```bash
λ> node
> const assert = require('assert')
undefined
> assert.equal('true', 'false')
AssertionError [ERR_ASSERTION]: 'true' == 'false'
    at repl:1:8
    at ContextifyScript.Script.runInThisContext (vm.js:50:33)
    at REPLServer.defaultEval (repl.js:240:29)
    at bound (domain.js:301:14)
    at REPLServer.runBound [as eval] (domain.js:314:12)
    at REPLServer.onLine (repl.js:441:10)
    at emitOne (events.js:120:20)
    at REPLServer.emit (events.js:210:7)
    at REPLServer.Interface._onLine (readline.js:282:10)
    at REPLServer.Interface._line (readline.js:631:8)
> assert.equal('true', 'true')
undefined
> 
```
Are you sure the tests are running? Could you provide more sample code?

**Comment by mingfang** [comment]
Assertions hangs for 30s when the assertion fails.
This example
```
Scenario('test', async(I) => {
  var assert = require('assert');
  //assert.equal('foo', 'foo'); //this works
  assert.equal('foo', 'bar'); //this does not work
});
```
fails with this error
```
Error: Evaluation timed out after 30000msec.  Are you calling done() or resolving your promises?
```

**Comment by reubenmiller** [comment]
The fix for this landed in v1.1.5.
